packages/css-framework/src/components/_transfer.scss

```diff
@@ -0,0 +1,59 @@
+.rn-transfer {
+  .column-left,
+  .column-right {
+    width: 40%;
+    float: left;
+    border: 1px solid #e0e8ed;
+    color: #012159;
```

Comment:
Colours should use the colour helpers to pick from the palette

Reply:
I seem to recall @hxltrhuxze telling me not to bother with variables as they are all about to change.

Reply:
All the more reason, you use the built in colours and you get the update for free